### PR TITLE
Do not run tests which are either broken or not applicable

### PR DESCRIPTION
**Pull request by ktf**
This limits the test being run to the actual ones which will
successfully run with the current, unmodified, alibuild setup.

### CODE REVIEW

**Comment by ktf** [comment]
@Barthelemy can you look into why the QC tests SIGABRT? Thanks.

File: Utilities/QC/QCMerger/CMakeLists.txt [hunk 1]
```diff
@@ -30,13 +30,15 @@ O2_GENERATE_EXECUTABLE(
...
+      test/MergerDeviceTestSuite.cxx
+      test/MergerTestSuite.cxx
+      )
+
+  O2_GENERATE_TESTS(
+      MODULE_LIBRARY_NAME ${LIBRARY_NAME}
+      BUCKET_NAME ${BUCKET_NAME}
+      TEST_SRCS ${TEST_SRCS}
+  )
+endif()
```
Comment (sawenzel) [inline Utilities/QC/QCMerger/CMakeLists.txt]: we should maybe add a ticket for each of these cases ... so as not to forget about them

File: DataFormats/Headers/src/DataHeader.cxx [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include <cstdio> // printf
 #include <cstring> // strncpy
 
+
 //the answer to life and everything
```
Comment (sawenzel) [inline DataFormats/Headers/src/DataHeader.cxx]: strange line. Is this a whitespace fix?

File: Utilities/QC/QCMerger/CMakeLists.txt (hunk 1 above)
Comment (ktf) [inline Utilities/QC/QCMerger/CMakeLists.txt]: Agreed. I am waiting for @Barthelemy to comment if I am missing anything obvious, though.

**Comment by Barthelemy** [comment]
I would have to check. The code had been written by student in Krakow. It is not really QC, it is mergers. I can do it tomorrow morning.

**Comment by ktf** [comment]
Ok. Let's wait until tomorrow and take a decision after you have checked.

File: Utilities/QC/QCMerger/CMakeLists.txt (hunk 1 above)
Comment (Barthelemy) [inline Utilities/QC/QCMerger/CMakeLists.txt]: It is a problem with DDS : https://alice.its.cern.ch/jira/browse/QC-7
I will contact Anar. You can disable the tests meanwhile.

Comment (ktf) [inline Utilities/QC/QCMerger/CMakeLists.txt]: Ok. Merging.

**Comment by sawenzel** [comment]
@ktf: Can we still add a ticket for each of the disabled unit tests ... so as not to forget about them?

**Comment by Barthelemy** [comment]
@sawenzel The ticket https://alice.its.cern.ch/jira/browse/QC-7 covers all these cases I think.

**Comment by sawenzel** [comment]
@Barthelemy : thanks ( however, I would honestly prefer to have purely software related stuff/bugs also visible on github ).

**Comment by Barthelemy** [comment]
I will see what is doable to see in github the JIRA tickets.

**Pull request merged**